### ISSUE TITLE
v1.0.0 te30/run0.sh and run.sh produce no graphs

### ISSUE THREAD

**Issue by trevorhardy** [pnnl/tesp#40]
Both `run0.sh` and `run.sh` run without error on the console but the graphing script produces an error and no graphs

```
$ python3 plots.py TE_Challenge


File TE_Challenge has baseMVA 100.0

Generator Dictionary:
Unit Bus Type Pnom Pmax Costs[Start Stop C2 C1 C0]
1 1 pv 163.0 247.0 [ 500.0 0.0 0.03 1.3 180.0 ]
2 2 swing 0.0 192.0 [ 2000.0 0.0 0.085 3.0 100.0 ]
3 3 pv 85.0 128.0 [ 3000.0 0.0 0.122 5.0 135.0 ]
4 9 pq 0.0 250.0 [ 3000.0 0.0 3.0 50.0 1500.0 ]
5 7 pq 0.0 0.0 [ 0.0 0.0 0.0 0.0 0.0 ]

DSO Bus Dictionary:
Bus Pnom Qnom ampFactor [GridLAB-D Substations]
7 100.0 35.0 250.0 ['SUBSTATION7']
Traceback (most recent call last):
  File "plots.py", line 14, in <module>
    pmetrics = pp.read_pypower_metrics (rootname)
  File "/usr/local/lib/python3.8/dist-packages/tesp_support/process_pypower.py", line 92, in read_pypower_metrics
    lst_b = json.loads(lp_b)
  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by temcdrm** [comment]
what os?

**Comment by trevorhardy** [comment]
I ran the TESP installer on a VirtualBox VM with a clean installation of Ubuntu LTS 20.04.1

**Comment by trevorhardy** [comment]
Still failing for me with the same error message after logout and login.

**Comment by temcdrm** [comment]
Cannot reproduce: Ubuntu 20.0.4 LTS, Python 3.8.5, tesp_support 0.9.4

Please zip up the contents of examples/te30 and post here

**Comment by trevorhardy** [comment]
Confirmed I'm still having problems; here's the .zip
[te30.zip](https://github.com/pnnl/tesp/files/5681997/te30.zip)

**Comment by temcdrm** [comment]
The simulations didn't actually run to completion.  _gridlabd.log_ from December 10 indicates an empty GLPATH causing error termination, and _*0.log_ from December 11 indicates those simulations never got started.

- When a simulation completes, none of the _*metrics.json_ files should have 0 byte sizes; there should at least be metadata in them
- If one simulation stops due to error, the other federates can remain in a hung state.  Use _ps_ or _lsof -i :5570_ to see which processes are still running on FNCS, or _lsof -i :23404.sh_ to see which processes are still running on HELICS.  Scrub with _./kill5570.sh_ or _./kill23404.sh_ as needed.  In FNCS, sometimes you have to use _kill -9 ####_ on a stubborn process, where _####_ is the PID from _ps_.
- There can't be any processes listening to 5570 (FNCS) or 23404 (HELICS) before starting a new case.  (Unless you manually reconfigure a case to use different ports.)

It's possible the December 10 failure was caused by the environment not being set. The December 11 failure could have been caused by stale processes left over from the day before.  To retry:

- _./clean.sh_
- _python3 prepare_case.py_
- _./run.sh_

**Comment by trevorhardy** [comment]
I restarted the VM (virtual power off and back on) and ran the above commands followed by `python3 plots.py TE_Challenge`: I still get the same error and am seeing the same empty output JSONs you referenced.

After trying the above when I look for hung processes us `ps` or `lsof` I don't see any.

```
> echo $GLPATH
/opt/tesp/lib/gridlabd:/opt/tesp/share/gridlabd
```

I've attached the lastest TE30 folder as well.
[te30.zip](https://github.com/pnnl/tesp/files/5690235/te30.zip)
